Microsoft/helium
# config var validation and secret fallback

Checks for required config inputs and exits if they don't exist.

Falls back to environment variables if they can't be retrieved from KeyVault.

To Test:

1. check out the code!
2. set all the necessary env vars to run the app normally
3. run the app, make sure the endpoints work
4. stop the app, unset the cosmosdb url env var
5. run the app, make sure it fails
6. reset the cosmosdb url env var
7. unset one of the keyvault vars (CLIENT_ID)
8. run the app, make sure it fails because it can't find the cosmosdb key or the app insights key
9. set the env vars for those secrets (see readme for their names)
10. run the app, make sure it works
